pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+  <modules>
+    <module>shell</module>
+    <module>credential</module>
+  </modules>
+
+  <parent>
+    <groupId>org.technologybrewery</groupId>
+    <artifactId>parent</artifactId>
+    <version>1</version>

```

Comment:
S: should be version 6, I think this will pull in most if not all of the plugin versions from the other comment